box.py

```diff
@@ -30,7 +30,7 @@
         yaml_support = False
 
 if sys.version_info >= (3, 0):
-    basestring = str
```

Comment:
This is a case for Python 3 specifically because `basestring` does not exist, make it a variable of `str` to check for `isinstance` below.